ansible_collections/arista/avd/roles/eos_designs/templates/network_services/vlan-interfaces.j2

```diff
@@ -96,12 +76,12 @@ vlan_interfaces:
 {%                     endfor %}
 {%                 endif %}
 {%             endif %}
-{%             if svi_raw_eos_cli is arista.avd.defined %}
+{%             if parent_profile.raw_eos_cli is arista.avd.defined %}
```

Comment:
Typo? Should be using `svi_settings`.
I think the accidental changes to existing molecule is caused by this.